app/javascript/react/src/containers/BookShowContainer.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import BookShowTile from '../components/BookShowTile'
+import ReviewTile from '../components/ReviewTile'
+
+
+class BookShowContainer extends React.Component {
+  constructor(props) {
+    super(props);
+      this.state = {
+        book: {},
+        reviews: []
+      }
+  }
+
+  componentDidMount() {
+    // let bookId = this.props.params.id;
+    fetch(`/api/v1/books/6`)
```

Comment:
Thanks! We don't have our routers working yet. This was just to test the showContainer to see if it was hooked up correctly. Do you still want us to get that set up, or wait to do it on another branch?